apps/ledger-live-desktop/src/env.js

```diff
@@ -2,5 +2,7 @@ import "dotenv/config";
 
 process.env = {
   ...process.env,
+  // FIXME: needed to build a workable test bundle of the app, to be removed one backend API is in prod
+  SWAP_API_BASE: "https://swap.aws.stg.ldg-tech.com/v3",
```

Comment:
It will stay until the Swap API is deployed in Prod (cf. https://ledgerhq.atlassian.net/browse/SRE-315)

It should be removed before being merged

Reply:
mmh can we use `live-common` env instead? I'm not sure this env approach will work properly across all envs (internal, renderer, main), even if you only need it at one place, i would recommend we move to the unified approach of env.ts because in that way we can see in one glance what are all our endpoints 🙏

Reply:
@gre indeed this `SWAP_API_BASE` is already in `env.ts` in `live-common`
Actually it was put here as well to be able to produce builds on the CI suing the staging env.

After reflexion, it can actually be removed now, the only issue is that the builds coming from the CI will not work "out of the box" until the API is deployed to prod (should happen today).
For testing the build, you will need to start the app with the above env var set manually

Reply:
Will remove